src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -1,10 +1,18 @@
+using System.ComponentModel;
 using FluentValidation;
 using NzbDrone.Core.Annotations;
 using NzbDrone.Core.ThingiProvider;
 using NzbDrone.Core.Validation;
-
 namespace NzbDrone.Core.Notifications.Telegram
 {
+    public enum MetadataLinkType
+    {
+        IMDb,
+        TVDb,
+        TVMaze,
+        Trakt,
```

Comment:
`Imdb` instead of a stylized format, use the `FieldOption` attribute to give it a formatted name, each value in the enum should have a integer assigned to it for future safetyy.

Take a look at `ReleaseType` to see how it's done there.